Sat Dec  4 01:25:18 EST 2010  Hamish Mackenzie <hamish.k.mackenzie@googlemail.com>
  * Add support for GHC 7
diff -rN -u old-ltk/ltk.cabal new-ltk/ltk.cabal
--- old-ltk/ltk.cabal	2010-12-30 14:29:13.066837842 +1100
+++ new-ltk/ltk.cabal	2010-12-30 14:29:13.068695088 +1100
@@ -6,13 +6,13 @@
 license-file: LICENSE
 copyright: Juergen "jutaro" Nicklisch-Franken
 maintainer: maintainer@leksah.org
-build-depends: Cabal >=1.6.0 && <1.9, base >=4.0.0.0 && <4.3,
-               containers >=0.2 && <0.4, filepath >=1.1.0 && <1.2,
-               glib >=0.10.0 && <0.12, gtk >=0.10.0 && <0.12,
-               mtl >=1.1.0.2 && <1.2, parsec >=2.1.0.1 && <2.2,
+build-depends: Cabal >=1.6.0 && <1.11, base >=4.0.0.0 && <4.4,
+               containers >=0.2 && <0.5, filepath >=1.1.0 && <1.3,
+               glib >=0.10.0 && <0.13, gtk >=0.10.0 && <0.13,
+               mtl >=1.1.0.2 && <2.1, parsec >=2.1.0.1 && <2.2,
                pretty >=1.0.1.0 && <1.1,
                -- Make sure we get the Cabal compaitble with ghc and haddock
-               haddock <2.8, ghc <6.13
+               haddock <2.9, ghc <7.1
 stability:
 homepage: http://www.leksah.org
 package-url: http://code.haskell.org/ltk
